imports/plugins/included/shipping-rates/server/no-meteor/schemas/restrictions.graphql

```diff
@@ -0,0 +1,217 @@
+####
+# Custom data types for FlatRateFulfillmentRestriction data
+####
+type attributeRestrictions {
+  property: String
+  value: String
+  propertyType: String
+  operator: String
+}
+
+type destinationRestrictions {
+  country: [String]
+  region: [String]
+  postal: [String]
+}
+
+enum restrictionTypeEnum {
+  allow
+  deny
+}
+
+enum FlatRateFulfillmentRestrictionSortByField {
+  createdAt
+}
+
+input attributeRestrictionsInput {
+  property: String
+  value: String
+  propertyType: String
+  operator: String
+}
+
+input destinationRestrictionsInput {
+  country: [String]
+  region: [String]
+  postal: [String]
+}
+
+
+####
+# Queries
+####
+extend type Query {
+  "Get the full list of flat rate fulfillment method restrictions."
+  getFlatRateFulfillmentRestrictions(
+    shopId: ID!
+    after: ConnectionCursor,
+    before: ConnectionCursor,
+    first: ConnectionLimitInt,
+    last: ConnectionLimitInt,
+    sortOrder: SortOrder = desc,
+    sortBy: FlatRateFulfillmentRestrictionSortByField = createdAt
+  ): FlatRateFulfillmentRestrictionConnection!
+
+  getFlatRateFulfillmentRestriction(
+    shopId: ID!
+    restrictionId: ID!
+    after: ConnectionCursor,
+    before: ConnectionCursor,
+    first: ConnectionLimitInt,
+    last: ConnectionLimitInt,
+    sortOrder: SortOrder = desc,
+    sortBy: FlatRateFulfillmentRestrictionSortByField = createdAt
+  ): FlatRateFulfillmentRestrictionConnection!
```

Comment:
This returns one document so should be

```
getFlatRateFulfillmentRestriction(restrictionId: ID!, shopId: ID!): FlatRateFulfillmentRestriction
```

- No connection args are needed because it's not a connection
- No exclamation on the response type because it might be `null` if the IDs you provide don't match anything.